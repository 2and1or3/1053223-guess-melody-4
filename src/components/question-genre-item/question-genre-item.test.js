import React from "react";
import renderer from "react-test-renderer";

import QuestionGenreItem from './question-genre-item.jsx';

it(`Render QuestionGenreItem component`, () => {
  const tree = renderer
    .create(
        <QuestionGenreItem
          genreAnswer = {`rock`}
          src = {`path`}
          id = {1}
          renderPlayer = {() => {}}
          userAnswer = {true}
          onChange = {() => {}}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
