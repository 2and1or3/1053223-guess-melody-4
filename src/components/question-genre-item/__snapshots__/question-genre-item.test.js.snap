// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render QuestionGenreItem component 1`] = `
<div
  className="track"
>
  <div
    className="game__answer"
  >
    <input
      checked={true}
      className="game__input visually-hidden"
      id="rock1"
      name="answer"
      onChange={[Function]}
      type="checkbox"
      value="rock"
    />
    <label
      className="game__check"
      htmlFor="rock1"
    >
      Отметить
    </label>
  </div>
</div>
`;
